### ISSUE TITLE
confused about how datasets work

### ISSUE THREAD

**Issue by skywo1f** [pjreddie/darknet#2061]
This is my .data file:
classes= 1
train  = train-run-less.txt
valid  = data/coco/5k.txt
names  = data/humans.names
backup = backup

I am trying to train just one class (person)  from the coco dataset. Something is not making sense, and in a sanity test, I discovered that the loss drops even if the training folder only contains 1 image (which doesnt even have people in it). I thought the way this worked was that it trained on the "train" images, then it tested the neural net on the "valid" images. How is it getting better at finding people in the "valid" images if it hasnt trained on a single one??
